### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by martaannaj**

### CODE REVIEW

**Review by florisbrunet — APPROVED** [review]

File: src/main/java/Area.java [hunk 1]
```diff
@@ -22,11 +22,17 @@ public String getName() {
...
+        StringBuilder finalDescription = new StringBuilder(description + "\n");
+        finalDescription.append(getItemsDescription());
+        return finalDescription.toString();
+    }
+
+    public String getItemsDescription() {
+        StringBuilder itemDescription = new StringBuilder("In this room, the following items are present: ");
         for (Item item: items) {
-            finalDescription.append("a <").append(item.getName()).append(">, ");
+            itemDescription.append("a <").append(item.getName()).append(">, ");
```
Comment (Edward2k) [inline src/main/java/Area.java:33]: nice new names!

File: src/main/java/BasicItem.java [hunk 2]
```diff
@@ -1,20 +1,8 @@
...
-        this.usedTo = usedTo;
-    }
-
-    public boolean isValidAction(String action) {
-        return true;
-    }
-
-    public boolean isValidManipulation(String action) {
-        return true;
+        super(name1, id, canBe, usedTo);
```
Comment (Edward2k) [inline src/main/java/BasicItem.java:6]: This simplifies it :O

Comment (Edward2k) [reply src/main/java/BasicItem.java:6]: But maybe if it is really just extending capabilites, we should make it Container extend BasicItem? And just omit the Abstract class

File: src/main/java/Game.java [hunk 3]
```diff
@@ -210,6 +214,25 @@ private static String handleMove(String direction, Player player) {
 		}
 	}
 
+	private static String handleContainer(String item, Player player, String action) {
```
Comment (Edward2k) [inline src/main/java/Game.java:223]: Good.

**Review by Edward2k — COMMENTED** [review]
Overall I think it is very good. The only thing to consider is if Container should be implementing the Abstract Item class, or we should omit the Item class and simply have BasicItem.
A pro to keeping the Item Abstract class is that it allows the client to add more types of interactibles easily. Maybe, for the sake of expandibility, it is best to keep it as is? (I think this but lemme know what you think). If we do choose this, we will have to document it. I'll be sure to add this to the Class diagram description part.

**Pull request merged**